assets/js/modules/analytics/components/setup/SetupFormUA.js

```diff
@@ -113,15 +118,17 @@ export default function SetupFormUA() {
 				</div>
 			) }
 
-			{ hasExistingUATag && <SetupUseSnippetSwitch /> }
+			{ hasExistingUATag && <SetupUseSnippetSwitchUA /> }
 
 			{ shouldShowGA4PropertyNotice && (
 				<GA4PropertyNotice
 					notice={ __(
 						'A Google Analytics 4 property will also be created.',
 						'google-site-kit'
 					) }
-				/>
+				>
+					{ hasExistingGA4Tag && <SetupUseSnippetSwitchGA4 /> }
+				</GA4PropertyNotice>
```

Comment:
The IB states:
>In assets/js/modules/analytics/components/setup/SetupFormUA.js:
If the existing UA tag is available, render the SetupUseSnippetSwitch component as a children prop within the GA4PropertyNotice component.

However, this is incorrect since the toggle snippet for the UA property has already been rendered above. As for the notice, we should render the toggle snippet for the GA4 property. Moreover, the notice text says, `A Google Analytics 4 property will also be created.`.